src/components/Checkbox/src/Checkbox.stories.tsx

```diff
@@ -29,41 +29,6 @@ const defaultCode = `
 </FormControlLabel>
```

Comment:
Shouldn't this be an mdx file (because of documentation purposes)?

Reply:
Now it is 😀